### DISCUSSION TITLE
ASUS UX393 seems to have inverted fan profiles

### DISCUSSION THREAD

**Discussion by StefanH1303** [seerge/g-helper#1209] [Q&A] [answered]
I really love G-Helper. Since I put it onto my ASUS UX393 Ultrabook, there is almost 1 GB RAM more free compared to before when all the ASUS services were still running (there is only the AsusNumPadService left now) and there is no 32-bit  process starting up anymore (Asus OSD is 32-bit only for some reason and always runs on Windows startup).
Anyway I am a bit confused by the choices that G-Helper offers. I only get the modes "Silent", "Balanced" and "Turbo". My ASUS BIOS does not accept new fan curves, so I have to stick with the old ones. But when I choose "Silent" the fan becomes really noisy. When I choose "Turbo" it becomes really silent.
Could it be that the naming is inversed? Otherwise I am always running with Turbo and CPU-Boost set to "Efficient Enabled" and the fan is almost always silent.

Does anyone have similar experiences with the CPU and fan behavior?

**Comment by seerge** [comment, marked as answer]
@StefanH1303 did you check https://github.com/seerge/g-helper/releases/tag/v0.116 ? There I swapped order of profiles for Vivobooks (it could be that yours work same)

**Reply by StefanH1303** [reply]
I normally only use Release versions, but in this case, it does not matter that much, I guess 😉
Actually the fans behave wit 0.116 exactly as I would have expected it. Silent is the slowest running one and Turbo immediately turns up the RPM. Thank you for the quick answer!

**Reply by seerge** [reply]
@StefanH1303 great :) glad to hear that Vivobook fix also applies to Zenbooks
